templates/github/.ci/scripts/docs-builder.py.j2

```diff
@@ -85,8 +88,17 @@ def main():
 
     ga_build = False
 
+    publish_at_root = False
+
     if (not re.search("[a-zA-Z]", branch) or "post" in branch) and len(branch.split(".")) > 2:
         ga_build = True
+        # Only publish docs at the root if this is the latest version
+        r = requests.get("https://pypi.org/pypi/{{ plugin_dash }}/json")
+        latest_version = version.parse(json.loads(r.text)["info"]["version"])
+        docs_version =  version.parse(branch)
```

Comment:
```suggestion
        docs_version = version.parse(branch)
```
It's correct that this branch-version still contains the post section, because it needs to compare to the package version on pypi, right?

Reply:
Yes, exactly.